components/Badge/index.js

```diff
@@ -17,6 +17,7 @@ const Badge = ({ id, image, name, description, loading = false }) => {
           loading ? (
             <Skeleton.Image className={styles.image} />
           ) : (
+            /* eslint-disable @next/next/no-img-element */
             <img className={styles.image} alt={description} src={image} />
```

Comment:
Why not use the Image component from next?

Reply:
These images come from the API, you can't do server optimizations because you only have the image in the client. Nevertheless, the bokken API already creates smaller versions of the images.